backend/src/govsg/middlewares/govsg-callback.middleware.ts

```diff
@@ -67,10 +64,6 @@ const parseWebhook = async (
     await GovsgCallbackService.parseWebhook(req.body, id)
     res.sendStatus(200)
   } catch (err) {
-    if (err instanceof UnexpectedWebhookError) {
-      res.sendStatus(500)
-      return
-    }
     if (err instanceof MessageIdNotFoundWebhookError) {
```

Comment:
Do you think we should discuss how we want our webhooks to behave? I'm not sure about the utility of returning non-200 statuses for cases such as invalid authentication or message id not being found if this will just trigger the same payload to get sent to us again. We can have a wider discussion with the other engineers on the team.

Reply:
> I'm not sure about the utility of returning non-200 statuses for cases such as invalid authentication

Thanks for catching that I missed out the `isAuthenticated`. On it.